benchmarks/memory/rmagick.gnuplot

```diff
@@ -0,0 +1,16 @@
+set datafile separator ","
+
+set key outside center top horizontal reverse Left samplen 2
+unset border
+set xtics scale 0
+set ytics scale 0
+set grid ytics linewidth 1 linetype -1
+
+set style line 1 lt 1 lc rgbcolor "#3465a4" lw 2.5 pt 7 ps 1
+set style line 2 lt 1 lc rgbcolor "#ff6347" lw 2.5 pt 7 ps 1
+set style line 3 lt 1 lc rgbcolor "#888a85" lw 2.5 pt 5 ps 1
+
+set xlabel "Nth loop"
+set ylabel "Memory usage (MiB)"
+
+plot "before.csv" with line linestyle 1, "after.csv" with line linestyle 2
```

Comment:
Can you share how this file is used? Should we have a `rake` task to automate it somehow? Or maybe we don't need it?

Reply:
1. Install `gnuplot`.
2. Retrieve two performance data to draw graph.
```
# Retrieve data before applying patches.
$ rake build
$ gem install pkg/rmagick-3.1.0.gem
$ cd benchmarks/memory
$ ruby image_new.rb > before.csv


# Apply the PR then retrieve improved data
$ rake build
$ gem install pkg/rmagick-3.1.0.gem
$ cd benchmarks/memory
$ ruby image_new.rb > after.csv
```

3. Launch `gnuplot` and execute `load 'rmagick.gnuplot'` command in prompt then the performance graph will be drew.
```
$ /usr/local/bin/gnuplot

	G N U P L O T
	Version 5.2 patchlevel 7    last modified 2019-05-29

	Copyright (C) 1986-1993, 1998, 2004, 2007-2018
	Thomas Williams, Colin Kelley and many others

	gnuplot home:     http://www.gnuplot.info
	faq, bugs, etc:   type "help FAQ"
	immediate help:   type "help"  (plot window: hit 'h')

Terminal type is now 'qt'
gnuplot> load 'rmagick.gnuplot'
```

Reply:
Added README how to draw the performance graph.

Reply:
Excellent, thanks!